css/main.css

```
@@ -1 +1,59 @@
-@import "tailwindcss";
\ No newline at end of file
+@import "tailwindcss";
+@import url("https://fonts.googleapis.com/css2?family=Syne:wght@400..800&display=swap");
+
+html {
+  width: 100%;
+  height: auto;
+  font-family: "Syne" !important;
+}
+@layer components {
+  .navbar-container {
+    max-width: 100%;
+    list-style-type: none;
+    align-items: center;
+    margin: 0 50px;
+    padding: 25px;
+    overflow: hidden;
+    display: flex;
+  }
```

Please use tailwind classes. See below example on how to use tailwind classes in a css file. 


```suggestion
   .navbar-container {  
        @apply w-full list-none flex items-center m-0 p-4 overflow-hidden;  
    }  
```